data-structures/src/data/structure/queue/overlinkedlist/LinkedQueue.java

```diff
@@ -0,0 +1,44 @@
+package data.structure.queue.overlinkedlist;
+
+import data.structure.linkedlist.LinkedList;
+import data.structure.map.overlinkedlist.LinkedHashMap;
+
+public class LinkedQueue<E> {
+    private int size;

```

Comment:
This field is unnecessary, as you're only writing it (lines 11, 17 and 21) but never reading it. Therefore, you can safely remove it.